configure

```diff
@@ -68,251 +68,7 @@ echo "$SWIG_PATH" > tensorflow/tools/swig/swig_path
 # Invoke python_config and set up symlinks to python includes
 (./util/python/python_config.sh --setup "$PYTHON_BIN_PATH";) || exit -1
 
-## Set up Cuda-related environment settings
-
-while [ "$TF_NEED_CUDA" == "" ]; do
```

Comment:
Sorry, missed this.  So, there are a couple problems with having fetch
create output dir symlinks. bazel-<workspace-name> is a sylink to the
execution root, which is created right before execution (not during the
loading phase, which is all that fetch runs). Also, it's sort of an
accident that the external repos are under bazel-<workspace-name> at all,
hopefully one that will be corrected in the next couple of days!

However, the point stands, it would be nice to see what the heck fetch
did.  I could see a couple possible options:

1. fetch could print paths to the repositories it fetched (that might be a
bit weird, because I think it would have to print their transitive closure
which might include repositories the user wasn't aware of).
2. Bazel could generally create a convenience symlink to external.

On Thu, Jul 28, 2016 at 5:07 AM, Damien Martin-guillerez <
dmarting@google.com> wrote:

> +Kristina Chodorow <kchodorow@google.com> to answer that question.
>
> On Thu, Jul 28, 2016 at 11:04 AM David Z. Chen <notifications@github.com>
> wrote:
>
>> In configure
>> <https://github.com/tensorflow/tensorflow/pull/3269#discussion_r72587457>
>> :
>>
>> > @@ -68,251 +68,7 @@ echo "$SWIG_PATH" > tensorflow/tools/swig/swig_path
>> >  # Invoke python_config and set up symlinks to python includes
>> >  (./util/python/python_config.sh --setup "$PYTHON_BIN_PATH";) || exit -1
>> >
>> > -## Set up Cuda-related environment settings
>> > -
>> > -while [ "$TF_NEED_CUDA" == "" ]; do
>>
>> Working on fixing these two issues.
>>
>> By the way, is there a reason why the fetch command doesn't create any
>> Bazel output dir symlinks? Seems that it might be useful to create the
>> bazel-<workspace-name> symlink at least so that we can look at the
>> external repositories that were fetched.
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/tensorflow/tensorflow/pull/3269/files/d744bfcabe598d7940a0d3bccf2d1b20feb229fe#r72587457>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/ADjHfz8aFtMMQCkQ8dY6IX9ZyuOjLpRwks5qaHCogaJpZM4JJPRy>
>> .
>>
>